packages/core/state/interaction/reducer.ts

```diff
@@ -195,6 +198,16 @@ export default makeReducer<InteractionStateBranch>(
             ...state,
             selectedPublicDataset: action.payload,
         }),
+        [SHOW_MOVE_FILE_MANIFEST]: (state, action: ShowMoveFileManifestAction) => ({
+            ...state,
+            visibleModal: ModalType.MoveFileManifest,
+            moveFileTarget: action.payload.target,
+        }),
+        [HIDE_VISIBLE_MODAL]: (state) => ({
+            ...state,
+            visibleModal: undefined,
```

Comment:
Confused about this. Was the visible modal otherwise never being set to `undefined`? Did you create this action in another PR?

Reply:
Whoops accidentally duplicated our existing. Resolved in [1e3c38b](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/1e3c38bfdd76d177129d05b25cc8fb37bc9aec84)